server/server.go

```diff
@@ -31,7 +32,12 @@ import (
 // Set is a Wire provider set that produces a *Server given the fields of
 // Options. This set might add new inputs over time, but they can always be the
 // zero value.
-var Set = wire.NewSet(New, Options{})
+var Set = wire.NewSet(
+	New,
+	Options{},
+	wire.Value(&DefaultDriver{}),
+	wire.Bind(new(driver.Server), new(*DefaultDriver)),
```

Comment:
As evidenced by [this build failure](https://travis-ci.com/google/go-cloud/jobs/151327402#L673), I led you astray. This line should be:

```go
wire.Bind(new(driver.Server), new(DefaultDriver)),
```